specification/redis/resource-manager/Microsoft.Cache/2017-02-01/redis.json

```diff
@@ -1369,10 +1369,7 @@
           "description": "Link for next set of locations."
         }
       },
-      "description": "The response of list firewall rules Redis operation.",
```

Comment:
@TimLovellSmith I discussed this with my peers and looks like even though we don't really do anything with a required property of a response model it does make sense to have it in the spec.
I am assuming that by marking this as `required` you intended to say that this property will always be sent by the server. If that is the case, feel free to undo the commit. Apologize for the confusion.

Reply:
Yup, that was the intention, but it seems that the linter likes it better without the attribute, so I think I'll just keep it as not required for now, and we can add it back later if there's ever a good reason to.

Reply:
Thanks, appreciate your patience 👍